calcom/cal.com
# fix: inconsistent uppercase issue resolved

Fixes #11115 

![image](https://github.com/calcom/cal.com/assets/126244804/76dba910-0359-4b6a-8ae0-c4332c6c0e64)
![image](https://github.com/calcom/cal.com/assets/126244804/fa3c2d4a-f634-4f47-a23d-1acc6835332a)

## How should this be tested?

Visit the getting-started page.
In step 2, you will see the issue resolved.

Complete all the steps and go to the profile settings page
Click on upload avatar, and you will see the issue resolved.

## Mandatory Tasks

- [ X] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.
